scripts/test-acceptance.sh

```diff
@@ -33,5 +33,15 @@ cd test/logs
 
 test_name="${1:-acceptance}"
 
+if [[ "${test_name}" = "accessibility" ]]; then
```

Comment:
The machines that these tests run on in jenkins have an old version of npm which doesn't seem to work with the requirements in this repo.  There is a devops ticket for this, but it may be better to wait until some of the other work they have going on now is done before attempting to install the custom rules via npm.      

This bit of code is a work around for that.  It will work both in jenkins and for local runs of the tests.